### PR TITLE
Esther fixing usa merge

### PR DESCRIPTION
**Pull request by estherrolf**
fixing script to merge USA policies and cases data, to reflect that the policies csv has the desired naming conventions now, so we don't have to rename columns :)

### CODE REVIEW

**Comment by estherrolf** [comment]
@bolliger32 i just realized im changing some of the cutil stuff, if you want me to change that back to the old ways lemme know

**Comment by bolliger32** [comment]
thanks! Yeah if you could change back that would allow people to run this from any working directory.

File: codes/data/usa/merge_policy_and_cases.py [hunk 1]
```diff
@@ -101,16 +101,17 @@ def min_no_nans(x):
 
 def download_and_process_policy_csv():
 
-	policy_data_raw = pd.read_csv(os.path.join(raw_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
+	policy_data_raw = pd.read_csv(os.path.join(int_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
```
Comment (bolliger32) [inline codes/data/usa/merge_policy_and_cases.py]: Note that this i/o will need to change to read from both `data/raw/USA_policy_data_sources.csv` and `data/interim/USA_policy_data_sources_other` and append these (as in the Policy Data section in #21 ). @ekrasovich and @jeanettelt can help advise on this.

File: codes/data/usa/merge_policy_and_cases.py [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 import numpy as np
 import os
-import codes.utils as cutil
 
 from functools import reduce
 
-raw_data_dir = cutil.DATA_RAW / 'usa'
-int_data_dir = cutil.DATA_INTERIM / 'usa'
-proc_data_dir = cutil.DATA_PROCESSED / 'adm1'
+raw_data_dir = '../../../data/raw/usa'
```
Comment (bolliger32) [inline codes/data/usa/merge_policy_and_cases.py]: Just a flag to change back, as mentioned in the conversation tab

File: codes/data/usa/merge_policy_and_cases.py [hunk 3]
```diff
@@ -101,16 +101,17 @@ def min_no_nans(x):
...
 
-	policy_data_raw = pd.read_csv(os.path.join(raw_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
+	policy_data_raw = pd.read_csv(os.path.join(int_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
 
 	# A. wrangle polices -> binary variables
 	# get the unchanged vars
 	policy_data = policy_data_raw.loc[:,['date', 'adm0_name', 'adm1_name', 'adm2_name']]
 	
-	policy_data.loc[:,'date'] = policy_data.loc[:,'date'].apply(fix_date)
+	# dates formatted as YYYY-MM-DD so no need to do this anymore
```
Comment (bolliger32) [inline codes/data/usa/merge_policy_and_cases.py]: Don't need to if you'd prefer commented, but should we just delete these lines now that we've got things formatted correctly. Feel free to resolve if you'd rather have them commented

File: codes/data/usa/merge_policy_and_cases.py (hunk 2 above)
Comment (bolliger32) [reply codes/data/usa/merge_policy_and_cases.py]: If you want these to be strings for later in the code, you could always just cast them to strings here

File: codes/data/usa/merge_policy_and_cases.py [hunk 4]
```diff
@@ -354,11 +357,12 @@ def add_countries(x):
 			cases_data_subset.drop([key], axis=1).reset_index(drop=True)
 
 	# add in population 
-	pops1 = pd.read_csv(os.path.join(str(int_data_dir).replace('usa','adm') , 'adm1/adm1.csv'), index_col = [0,1])
+	pops1 = pd.read_csv(os.path.join(int_data_dir.replace('usa','adm') , 'adm1/adm1.csv'), index_col = [0,1])
```
Comment (bolliger32) [inline codes/data/usa/merge_policy_and_cases.py]: If we do go with the cutil Path objects, will need to either cast them to str at the top of the script or here.

File: codes/data/usa/merge_policy_and_cases.py (hunk 1 above)
Comment (estherrolf) [reply codes/data/usa/merge_policy_and_cases.py]: yeah i can do this on a future PR!

File: codes/data/usa/merge_policy_and_cases.py (hunk 2 above)
Comment (estherrolf) [reply codes/data/usa/merge_policy_and_cases.py]: ok fixed that on the modified version

File: codes/data/usa/merge_policy_and_cases.py (hunk 3 above)
Comment (estherrolf) [reply codes/data/usa/merge_policy_and_cases.py]: i can remove them on the next PR; trying to get it so we can rerun the data rn

File: codes/data/usa/merge_policy_and_cases.py (hunk 4 above)
Comment (estherrolf) [reply codes/data/usa/merge_policy_and_cases.py]: i cast them at the top

**Pull request merged**